api.json

```diff
@@ -30,13 +30,18 @@
 			"ERR_BUG": 47,
 			"ERR_BUSY": 44,
 			"ERR_CANT_ACQUIRE_RESOURCE": 28,
+			"ERR_CANT_CONNECT": 25,
```

Comment:
@BastiaanOlij Wasn't it intended that enum values get ordered by names instead of their value?